src/graph_compute/numa_scheduling.h

```diff
@@ -183,14 +183,16 @@ static inline void createChunkData(vertex_t * const nodes,
   scheddata->numChunksPerWorker =
     (scheddata->cntChunks + NUMA_WORKERS - 1) / NUMA_WORKERS;
 
-  for (vid_t i = 0; i < scheddata->cntChunks; ++i) {
+  cilk_for (vid_t i = 0; i < scheddata->cntChunks; ++i) {
```

Comment:
Are we using Cilk in NUMA? I thought we were avoiding it as a precaution against weird interactions with pthreads.